packages/slate-editor/src/components/EditorBlock/Menu.tsx

```diff
@@ -103,9 +33,9 @@ export function Menu({ children, className, onClick, popperOptions, reference }:
     return (
         <Popper
             referenceElement={reference}
-            modifiers={getModifiers(popperOptions)}
-            placement={placement}
+            modifiers={getMenuPopperModifiers(popperOptions)}
             strategy="fixed"
+            placement={placement}
```

Comment:
ESLint should've complained about the prop order on this one 🤔

Reply:
Looks like we do not have such rule enabled here, i will do that in separate PR since there will be a lot of changes

Reply:
Then nevermind, I though we have the same rules enabled here